components/org.wso2.carbon.identity.sso.agent/src/main/java/org/wso2/carbon/identity/sso/agent/SSOAgentConstants.java

```diff
@@ -70,6 +70,8 @@ private SSOAgentConstants() {
         public static final String OAUTH2_SAML2_GRANT_URL = "OAuth2SAML2GrantURL";
         public static final String SKIP_URIS = "SkipURIs";
         public static final String QUERY_PARAMS = "QueryParams";
+        public static final String Secretkey_FilePath = "/home/nilasini/Documents/tomcat/apache-tomcat-8.0.45/webapps/travelocity" +
+                ".com/WEB-INF/classes/temp_file.txt";
```

Comment:
"temp_file" is not a good name. Does not implies the related context. Please use something similar to secure vault.